lively.morphic/components/policy.js

```diff
@@ -1052,7 +1052,7 @@ export class StylePolicy {
    */
   synthesizeSubSpec (submorphNameInPolicyContext, ownerOfScope, previousTarget) {
     const isRoot = !submorphNameInPolicyContext;
-    const transformProps = ['extent', 'position', 'rotation', 'scale', 'lineHeight'];
+    const transformProps = ['extent', 'position', 'scale', 'lineHeight'];
```

Comment:
"apply rotation for root of component for now" - why only 'for now'?

Reply:
Because I am not yet sure if we run into unintended consequences. The reason I excluded transform properties from being applied to a morph when it is *the root* is that we mostly dont want that, even if these properties are not explicitly overridden.
For instance if we do not exclude the position from being applied to the root, it will lead to weird jump behaviors of morphs that suddenly align at positions that make no sense in the context they are in. For now I assume that rotation is seldomly defined at all, so it wont cause any issue. Also since it is applied so rarely you would assume that *if* a rotation is applied, it is there for a good reason.